### PR TITLE
add mael/intervention-image-bundle

### PR DESCRIPTION
**Pull request by Mael-91**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by Mael-91** [comment]
> Pull request does not pass validation.

I don't see where I made a mistake?

**Comment by Mael-91** [comment]
I'm sorry, I got the commits file wrong.

File: mael/intervention-image-bundle/1.0/post-install.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<bg=blue;fg=white> Next for MaelInterventionImageBundle </>
+    * <fg=blue>Edit</> mael_intervention_image.yaml to configure the driver in  config/packages
```
Comment (Nyholm) [inline mael/intervention-image-bundle/1.0/post-install.txt:2]:
```suggestion
    * <fg=blue>Edit</> mael_intervention_image.yaml to configure the driver in config/packages
```

**Comment by Mael-91** [comment]
I just committed the correction, thank you!

**Pull request merged**